timers.ts

```diff
@@ -92,3 +87,5 @@ export function clearTimer(id: number) {
     timerClearId: id
   });
 }
+
```

Comment:
`no-multiple-empty-lines`

Reply:
Fixed in latest commit :)

Reply:
@yorkie you still have `eol-last` (`Consecutive blank lines are forbidden`)
https://travis-ci.com/ry/deno/builds/75008749#L5323